kits/rosie/rosie_demo.cpp

```diff
@@ -1,4 +1,5 @@
 #define EIGEN_INITIALIZE_MATRICES_BY_NAN
```

Comment:
Why should we do this by default?

Reply:
oh I dont need this anymore, I'll take it out